### PR TITLE
Gui draft

### PR DESCRIPTION
**Pull request by snafees**
Many thanks to contributing to snafees/ortho_seqs!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [x] Ensure the test suite passes with [pytest](https://docs.pytest.org/en/latest/) . (command to run: `pytest` or `make coverage` if you want to see which lines don't have tests yet)
 - [x] Make sure your code is linted and autoformatted using [black](https://github.com/psf/black) (`black . --check`).
 - [ ] Documentation in `usage.md` is updated
 - [x] `README.md` is updated

This PR is to put out a draft version of the GUI skeleton.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
     description="A PyPI package to compute multivariate tensor-based orthogonal polynomials for sequence data and map phenotypes onto sequence space.",
-    name="ortho_seq_code",
+    name="ortho_seqs",
     version="1.0.1",
     url="https://github.com/snafees/ortho_seqs",
     packages=find_packages(),
     keywords=[
         "regression",
-        "covariance",
+        "projections" "covariance",
```
Comment (AhmetCanSolak) [inline setup.py:16]: we are missing a comma and a newline here I believe

File: ortho_seq_code/gui.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+        self.widget_layout.addWidget(start_button)
+        self.setLayout(self.widget_layout)
+
+
+class MainWindow(QMainWindow):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.setCentralWidget(MainWidget())
+        self.originalPalette = QApplication.palette()
```
Comment (AhmetCanSolak) [inline ortho_seq_code/gui.py:81]: this is not used it seems, maybe we can clean it?

**Review by AhmetCanSolak — APPROVED** [review]
I dropped couple little comments but overall looks good, we also need to update this branch and resolve the conflict but then we can merge. Thanks @snafees !

File: setup.py (hunk 1 above)
Comment (snafees) [reply setup.py:16]: yes, thank you, fixing it in another branch but will fix it here as well :)

File: ortho_seq_code/gui.py (hunk 2 above)
Comment (snafees) [reply ortho_seq_code/gui.py]: hmm, that's right... I've commented out line 81

Comment (snafees) [reply ortho_seq_code/gui.py]: removed it for now!

**Pull request merged**